### PR TITLE
CAL-1335: Add more times for event reminder

### PR DESCRIPTION
**Pull request by wkhessairi**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less [hunk 1]
```diff
@@ -124,12 +124,19 @@
 
 
 /*===================UIEventRemindersTab===========================*/
+.reminderInput {
+	float: left;
+	input {
+		width: 60px;
+		margin-right: 5px;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less]: The style of the number field is correct, see 
image

Comment (wkhessairi) [reply platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less]: I didn't get the point. You mean that we should not display arrows (up and down) ?

Comment (thomasdelhomenie) [reply platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less]: No, I mean we should remove the little space on the right-side of the arrows.

Comment (wkhessairi) [reply platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less]: okay :+1:

File: platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less [hunk 2]
```diff
@@ -124,12 +124,22 @@
...
+	}
+	input[type=number] {
+		padding-right: 2px;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less]: Why do we set a padding-right to 2px ? It creates a little space on the right : 
image
I would set it to 0. WDYT ?

Comment (wkhessairi) [reply platform-ui-skin/src/main/webapp/skin/less/calendar/uiPopup/Style.less]: okay

**Pull request merged**